new_study_assistant.py

```diff
@@ -0,0 +1,624 @@
+        f.write('Predictions are given as to whether an attribute is a patient or sample attribute.\n')
+        f.write('  The sample/patient prediction is based on what is most common for that particular attribute in the existing cBioPortal studies.\n')
+        f.write('<br>\n')
+        f.write(predicted_types.to_html(index=False)) 
+        f.write('<br>\n')    
+
+    #with doc.create(Subsection('Number of attributes distribution\n')):
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_attribute_distribution.png\"' + 'alt=\"number of attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of attributes in the new study and the number of attributes in each existing study on cBioPortal.  The dashed black line indicates the number of attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.write('<br>')
+
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_unique_attribute_distribution.png\"' + 'alt=\"number of unique attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of unique attributes in the new study and the number of unique attributes in each existing study on cBioPortal.  The dashed black line indicates the number of unqiue attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.write('<br>')
+
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_common_attribute_distribution.png\"' + 'alt=\"number of common attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of common attributes in the new study and the number of common attributes in each existing study on cBioPortal.  The dashed black line indicates the number of common attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.close()
+
+#values that should be read in
+# Parse arguments
+file_directory=''
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--new_study_path", help="path to new study data")
+parser.add_argument("--study_to_drop", help="if study being tested is already on cBioPortal it may be helpful to drop that study from the analysis")
+parser.add_argument("--specific_study", help="name of specific study on the portal to test")
+parser.add_argument("--datahub_path", help="directory where the datahub is located")
+parser.add_argument("--output_pdf", action='store_true', help="flag to output pdf")
+args = parser.parse_args()
```

Comment:
good argument handling. There could be a summary of arguments up top in comments. Also, I don't see code here to output a summary of arguments to users (usage), but maybe that is a hidden function of the parse_args() function if the user provides any unrecognized options. (like --help)